op-e2e/actions/interop/interop_txplan_test.go

```diff
@@ -71,6 +71,7 @@ func TestTxPlanDeployEventLogger(gt *testing.T) {
 	is := dsl.SetupInterop(t)
 	actors := is.CreateActors()
 	actors.PrepareChainState(t)
+	actors.VerifyInitialState(t)
```

Comment:
they seem to be called almost always together, so maybe add a `PrepareAndVerifyChainState` helper that calls both?

Reply:
Fixed in c6fd3b743583a4bdbc626c27c8e5b1a8af210d24.